models/modelsmodule.cpp

```diff
@@ -115,37 +115,37 @@
 #include "weight_recorder.h"
 
 // Prototypes for synapses
-#include "bernoulli_connection.h"
-#include "clopath_connection.h"
+#include "bernoulli_synapse.h"
+#include "clopath_synapse.h"
 #include "common_synapse_properties.h"
-#include "cont_delay_connection.h"
-#include "cont_delay_connection_impl.h"
+#include "cont_delay_synapse.h"
+#include "cont_delay_synapse_impl.h"
 #include "diffusion_connection.h"
 #include "gap_junction.h"
-#include "ht_connection.h"
-#include "jonke_connection.h"
-#include "quantal_stp_connection.h"
-#include "quantal_stp_connection_impl.h"
+#include "ht_synapse.h"
+#include "jonke_synapse.h"
+#include "quantal_stp_synapse.h"
+#include "quantal_stp_synapse_impl.h"
 #include "rate_connection_delayed.h"
 #include "rate_connection_instantaneous.h"
 #include "spike_dilutor.h"
```

Comment:
Shouldn't the `spike_dilutor.h` be imported under `// Stimulation devices` or `// Neuron models`?

Reply:
Added under devices, as the spike_dilutor class inherits from DeviceNode.